Automattic/woocommerce-payments
# Fix alignment issue for the warning icon in settings.

Fixes #9304 

#### Changes proposed in this Pull Request
As mentioned on #9304, certain versions of Gutenberg cause an alignment issue when the warning icon is shown beside the WooPay or Link payment methods on the settings screen. This issue can be reproduced by installing the Gutenberg plugin on your local environment and enabling the WooPay or Link payment methods.

**Before:**
<img width="600" alt="existing-issue" src="https://github.com/user-attachments/assets/d612d0ca-1c8c-44dc-8584-b1516e0af5d1">

**After with default Gutenberg version**
<img width="600" alt="after-with-current-gutenberg" src="https://github.com/user-attachments/assets/3dc1aff7-4895-41ea-982e-99b8a2bc1c7c">

**After with a newer Gutenberg version**
<img width="600" alt="after-with-newer-gutenberg" src="https://github.com/user-attachments/assets/231fbec5-f80b-41c0-b1dd-9d248327552c">

#### Testing instructions
* Install and activate the "Gutenberg" plugin.
* Before applying this PR, visit the WooPayments settings screen.
* Check the "WooPay" or "Link" boxes and observe the logo alignment issue as noted above.
* Apply this PR and visit the WooPayments settings screen.
* Check the "WooPay" or "Link" boxes and observe that the logo alignment issue no longer occurs
* Disable the "Gutenberg" plugin.
* Visit the WooPayments settings screen.
* Check the "WooPay" or "Link" boxes and observe that the logo alignment issue still does not occur.

-------------------


**Post merge**
